graphql/resolvers.go

```diff
@@ -1961,7 +1961,10 @@ func (r *queryResolver) BuildBaron(ctx context.Context, taskId string, exec int)
 	execString := strconv.Itoa(exec)
 	searchReturnInfo, projectNotFound, err := GetSearchReturnInfo(taskId, execString)
 	if projectNotFound {
-		return nil, ResourceNotFound.Send(ctx, err.Error())
+		return &BuildBaron{
+			SearchReturnInfo:     nil,
+			BuildBaronConfigured: !projectNotFound,
```

Comment:
this will be easier to read if we set BuildBaronConfigured to `false` instead of `!projectNotFound`

Reply:
I've thought about it, but I don't want to hardcode it... especially since I use BuildBaronConfigured: !projectNotFound, later in the function too, so I don't want to hardcode true and false.